cfme/containers/provider/openshift.py

```diff
@@ -138,6 +125,23 @@ def __init__(
             endpoints=endpoints,
             appliance=appliance)
 
+    def create(self, **kwargs):
+
+        # Enable alerts collection before adding the provider to avoid missing active
+        # alert after adding the provider
+        # For more info: https://bugzilla.redhat.com/show_bug.cgi?id=1514950
+        if self.appliance.version >= '5.9' and getattr(self, "alerts_type") == "Prometheus":
```

Comment:
__REQUIRED__
If you don't specify a default in the ```getattr``` and `alert_type` is missing it'll raise an `AttributeError`. please add a default, i.e.
```python
if self.appliance.version >= '5.9' and getattr(self, "alerts_type", None) == "Prometheus":
```

Reply:
If "alert_type"  missing then the statement return False, I don't see why an exception is required here.

Reply:
And default is ```None``` in this case

Reply:
I added a default before but @mshriver asked me to delete it for some reason.

Reply:
@mshriver Do you want me to add back the default value?

Reply:
As far as I remember the reason for default removal is to get this exception, so we will fix the yamls accordingly